frontend/src/components/Footer/footer.css

```diff
@@ -1,8 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=Rethink+Sans:ital,wght@0,400..800;1,400..800&display=swap');
+:root {
+  --text-font: 'Rethink Sans', sans-serif;
+  --primary: #fff;
+  --footer-bg-color: #1a1c24;
+  --footer-text-color: #e7ebf5;
+  --card-bg-color: #0b0c10;
+  --footer-line-color: rgba(231, 235, 245, 0.2);
+}
+
+body {
+  overflow-x: hidden;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style-type: none;
+}
+
```

Comment:
this seems to partially repeat functionality that already exists in `globals.css`, please combine missing parts into `globals.css` and remove this from `footer.css` and verify everything still works correctly